samples/nrf9160/lwm2m_client/src/lwm2m/include/lwm2m_app_utils.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <net/lwm2m.h>
+#include <drivers/sensor.h>
+
+#ifndef LWM2M_APP_UTILS_H__
+#define LWM2M_APP_UTILS_H__
+
+/* LwM2M IPSO Object IDs */
+#define IPSO_OBJECT_SECURITY_ID 0
+#define IPSO_OBJECT_SERVER_ID 1
+
+#define IPSO_OBJECT_DEVICE_ID 3
+
+#define IPSO_OBJECT_COLOUR_ID 3335
+
+/* Server RIDs */
+#define LIFETIME_RID 1
+
+/* Device RIDs */
+#define MANUFACTURER_RID 0
+#define MODEL_NUMBER_RID 1
+#define SERIAL_NUMBER_RID 2
+#define FACTORY_RESET_RID 5
+#define POWER_SOURCE_RID 6
+#define POWER_SOURCE_VOLTAGE_RID 7
+#define POWER_SOURCE_CURRENT_RID 8
+#define CURRENT_TIME_RID 13
+#define DEVICE_TYPE_RID 17
+#define HARDWARE_VERSION_RID 18
+#define BATTERY_STATUS_RID 20
+#define MEMORY_TOTAL_RID 21
+
+/* Location RIDs */
+#define LATITUDE_RID 0
+#define LONGITUDE_RID 1
+#define ALTITUDE_RID 2
+#define LOCATION_RADIUS_RID 3
+#define LOCATION_VELOCITY_RID 4
+#define LOCATION_TIMESTAMP_RID 5
+#define LOCATION_SPEED_RID 6
+
+/* Misc */
```

Comment:
Being miscellaneous -> you could describe the purpose of each of the macros

Reply:
Done